image-analysis/api/README.md

```diff
@@ -0,0 +1,413 @@
+<!DOCTYPE html>
```

Comment:
I think this file extension is wrong -- this is html?

Reply:
This was in the existing READMEs too. I think GitHub might like READMEs to be .md files even if they are actually html. Maybe it won't display properly with an html extension.